src/main/java/fishman/exception/FishmanException.java

```diff
@@ -0,0 +1,148 @@
+package fishman.exception;
+
+/**
+ * Represents the base exception class for Fishman bot custom exception
+ */
+public class FishmanException extends Exception {
+
+    /**
+     * Constructs a new FishmanException with no message.
+     */
+    public FishmanException() {
+    }
+
+    /**
+     * Constructs a new FishmanException with the specified message
+     *
+     * @param message The detail message.
+     */
+    public FishmanException(String message) {
```

Comment:
Nice naming of all exception classes, it is clear what each one is for

Reply:
Exception class naming is really well done here